.travis.yml

```diff
@@ -3,8 +3,9 @@ os:
     - linux
 osx_image:  xcode12.2
 python:
-    - "2.7"
     - "3.7"
+    - "3.8"
+    - "3.9"
 install: 
     - pip install -r requirements.txt
     - pip install pyoscode==1.0.2
```

Comment:
Wait isn't `pip install pyoscode==1.0.2` problematic? That won't install the current version (the one that is supposed to be tested)... Shouldn't this be `pip install .`?

Reply:
Yes, thank you for spotting that!